AnimationGroup.lua

```diff
@@ -128,8 +128,11 @@ function AG:LoadProperties(group)
     group.parent:SetHeight(group.properties.height)
 
     local point = group.properties.point
-    if point and point[1] then
-        group.parent:SetPoint(unpack(point))
+        -- old implementation some times fires error =)
+	if point and type(point) == "table" then  
+		local point, region ,relativePoint, offsetX, offsetY = unpack(point)
+		region = type(region) ~= "function" and region or nil 
+        group.parent:SetPoint(point, region ,relativePoint, offsetX, offsetY)
```

Comment:
```suggestion
    if point and type(point) == "table" then  
	      local point, relative_to, relative_point, offset_x, offset_y = unpack(point)
	      relative_to = type(relative_to) ~= "function" and relative_to or nil 
        group.parent:SetPoint(point, relative_to, relative_point, offset_x, offset_y)
```

Why filter all cases where `relative_to` refers to a region or a frame? As far as I can tell, this only allows global frame references by name, e.g. `'PlayerFrame'`. 

This function should be a no-op for the first call, because is just recovers the initial position that it had on creation.

Obviously, if the initial `relative_to` reference was reclaimed, I could imagine its value would be invalid. However, to my knowledge frames cannot be garbage collected.

Can you point at a particular case where you get a problematic region/relative_to value?

Reply:
Sometimes "region" UIParent from GetPoint returns like function - didnt get why. May be some bugs in client.
Also need check on lengh of point = i got errors if group.parent dont have any and point was like {} in OmniCC =)